android/testng-examples/src/test/resources/com/browserstack/run_first_test/first.conf.json

```diff
@@ -4,15 +4,17 @@
   "key": "BROWSERSTACK_ACCESS_KEY",
```

Comment:
Can we change `user` to `username` and `key` to `access_key` to make it uniform in all code sample repos.
e.g https://github.com/kn-neeraj/python-appium-app-browserstack/blob/python3-readme-changes/android/examples/run-first-test/config.json